start-aws.py

```diff
@@ -0,0 +1,62 @@
+import os
+import time
+
+import boto.ec2
+
+
+# AWS region
+aws_region = 'us-west-2'
+
+# Name of security group exposing ports for Buildmaster
+security_group = 'Buildbot staging'
```

Comment:
This script is very specific to how the ClusterHQ AWS account is set up right now. Either document how to set up a security group / key name etc so this works or automate setting up those things.

I know that the security group documentation is kinda-sorta in the README but the connection should be explicit.